### PR TITLE
docs: rewrite reconfigure renovate

### PR DESCRIPTION
**Pull request by HonkingGoose**
## Changes:

Basically a full rewrite.

Major changes:

- Use lists for steps
- Use present tense, active voice
- Add command to install `renovate` globally with Yarn package manager
- Move CircleCI orb section to separate `h2` on bottom of page
- Change page description

## Context:

Helps with issue #7626.

## Documentation (please check one with an [x])

- [x] I have updated the documentation, or
- [ ] No documentation update is required

## How I've tested my work (please tick one)

I have verified these changes via:

- [x] Code inspection only, or
- [ ] Newly added unit tests, or
- [ ] No new tests but ran on a real repository, or
- [ ] Both unit tests + ran on a real repository

### CODE REVIEW

File: docs/usage/reconfigure-renovate.md [hunk 1]
```diff
@@ -1,22 +1,41 @@
...
 
-If you wish to make config edits directly, it's recommended to do so via a PR and then run Renovate's config validator to verify it.
-The validator is named `renovate-config-validator` and installed alongside `renovate` itself if you run `npm i -g renovate` or equivalent.
-If it validates your new config then it should be safe to merge.
+If you want to make config edits directly, follow these steps:
+
+1. Create a new Git branch to work on
+1. Install the `renovate` package globally (`npm i -g renovate` or `yarn global add renovate`) to get the `renovate-config-validator` program
+1. Edit your Renovate configuration file
+1. Validate your config by running `insert proper command here`
```
Comment (HonkingGoose) [inline docs/usage/reconfigure-renovate.md:21]: I don't know what the proper command is to run the validator (as I've never used it...), so please suggest the proper command. 😄

File: docs/usage/reconfigure-renovate.md [hunk 2]
```diff
@@ -1,22 +1,41 @@
 ---
 title: Reconfiguring Renovate
```
Comment (HonkingGoose) [inline docs/usage/reconfigure-renovate.md:2]: The `title: Reconfiguring Renovate` and the filename `docs/usage/reconfigure-renovate.md` don't  match, do we want to fix that?

Comment (rarkins) [reply docs/usage/reconfigure-renovate.md:2]: I don't think it's necessary

**Review by rarkins — DISMISSED** [review]

File: docs/usage/reconfigure-renovate.md (hunk 1 above)
Comment (HonkingGoose) [reply docs/usage/reconfigure-renovate.md:21]: @rarkins Have you seen this comment? We still need to add the proper command. 😄

Comment (rarkins) [reply docs/usage/reconfigure-renovate.md:21]: It's `renovate-config-validator`

Comment (HonkingGoose) [reply docs/usage/reconfigure-renovate.md:21]: I'll make the commit to fix the command. 😄

**Comment by HonkingGoose** [comment]
@rarkins This should be good to go now. 😄 
Sorry for pulling on the emergency cord... 😉

**Pull request merged**